cc3d/core/GraphicsOffScreen/MVCDrawModel2D.py

```diff
@@ -317,42 +260,39 @@ def init_vector_field_actors(self, actor_specs, drawing_params=None):
         vectors_int_addr = extract_address_int_from_vtk_object(vtkObj=vectors)
 
         fill_successful = False
-        # lattice_type_str = self.get_lattice_type_str()
-        #
-        # if lattice_type_str.lower() == 'hexagonal' and drawing_params.plane.lower() == "xy":
         if self.is_lattice_hex(drawing_params=drawing_params):
-            if field_type == 'vectorfield':
+            if field_type == "vectorfield":
```

Comment:
We can but let's do it in the later version. I really want to close things off and get the release going - I am afraid this might be a bigger job than just changing it here - plater and twedit might need to be changed as well. I created the issue:

https://github.com/CompuCell3D/CompuCell3D/issues/633

Reply:
Works for me